pingPong/schema.sql

```diff
@@ -0,0 +1,13 @@
+create table users(
+    id serial primary key,
+    name text
+);
+
+create table games(
+    id serial primary key,
+    player1 text, 
```

Comment:
You might not need the `player1` and `player2` fields, since you have foreign keys to them in this table.